docs/user-guide/docs-source/appendix-contributing.md

```diff
@@ -24,7 +24,13 @@ Launch your sample project by clicking on the `*.uproject` file. When prompted t
 
 ![Missing Modules prompt](images/appendix/RebuildMissingModules.png)
 
-Once the project opens, enable the "Functional Testing Editor" plugin, and restart (or simply close) the editor for the plugin to take effect. This plugin is used to run Ambit's automated tests.
+Next, you'll need to enable some plugins to support running automated tests. Once the project opens, enable the following plugins:
+
+- Functional Testing Editor
+- Runtime Tests
+- Editor Tests
+
+Restart (or simply close) the editor for the plugins to take effect.
```

Comment:
Minor: Do you see a reason to specify that the user should look to enable these plugins in the `Edit > Plugins` window, or are you expecting them to know how to do this after reading the one-time setup guide?

Reply:
Fair enough